Alan_DCMotorTest/Motor_code.ino

```diff
@@ -0,0 +1,43 @@
+#include <Wire.h>
+#include <Adafruit_MotorShield.h>
+#include "utility/Adafruit_MS_PWMServoDriver.h"
+
+// Create the motor shield object with the default I2C address
+Adafruit_MotorShield AFMS = Adafruit_MotorShield(); 
+
+// Connect two motors with the following terminals
+Adafruit_DCMotor *motor1 = AFMS.getMotor(1); // Motor connected to M1
+Adafruit_DCMotor *motor2 = AFMS.getMotor(2); // Motor connected to M2
+Adafruit_DCMotor *motor3 = AFMS.getMotor(3); // Motor connected to M3
```

Comment:
For now we won't need to deal with a third motor since Samruddhi and I are yet to design the compression hub, so if you want you can add another motor (4 motors total) to prepare for that.